### ISSUE TITLE
Docker run command support

### ISSUE THREAD

**Issue by ajurasz** [hekonsek/zed#57]
We can start docker container without issue but this is not enough we should also support run command because even if I can start mongodb with our deployer I cannot specyfi how to start it, so I suggest to add support to at least following run command options:
<code>-d</code> to run in the background
<code>-p</code> publishing container's port to the host
<code>-v</code> bind mount a volume

so I could do something like this:

deploy_start <deploymentId> -d -p 3000:27017

**Comment by hekonsek** [comment]
We should start by adding options support to the deployment URI. For example:

    deploy docker:repo/image:tag?d=true&v=/foo/bar:/baz/qux&p=8181:8181

This information will be saved to the deployment descriptor and therefore executing `deploy_start id` or `deploy_start_all` will start the image with predefined configuration. Keep in mind that port mappings or volumes configuration should be specified once, at the deployment time. Otherwise you will be forced to specify them every time your start the deployable :) .

As a bonus, we can add support for overriding these settings at the `deploy_start` level, but first of all we need to add them to the deployment URI.

Does my explanation make sense? :)

**Comment by ajurasz** [comment]
Yes, keeping these information in deployment descriptor give us more flexibility and make it more user friendly in opposed to what I suggested.

**Comment by hekonsek** [comment]
However keeping mind that the option to override some deployable URI settings on the start level is a useful feature as well. Especially for the production environment - if you need to restart container with some temporary settings changed :) .

Can you create dedicated issue for this idea?

**Comment by ajurasz** [comment]
Done (#58).

**Comment by ajurasz** [comment]
@hekonsek starting to work on this.

**Comment by hekonsek** [comment]
Maybe you would like to take a look at #59 instead? It's more important IMHO :) .

**Comment by ajurasz** [comment]
Sure I will look at #59 in the first place, no problem.
